### PR TITLE
Add Python installation and its dev tools.

### PR DESCRIPTION
**Pull request by shank7485**

### CODE REVIEW

File: tests/test_functions [hunk 1]
```diff
@@ -93,6 +93,13 @@ function test_install_nodejs {
     asserts_file_exist /usr/bin/npm
 }
 
+# test_install_python() - Verify the correct installation of Python
+function test_install_python {
+    install_python
+    asserts_installed_package python2.7
+    asserts_file_exist /usr/bin/python
```
Comment (electrocucaracha) [inline tests/test_functions]: The line 100 should validate the tox binary, given that this is not installed via apt-get

File: lib/functions [hunk 2]
```diff
@@ -118,6 +118,14 @@ function install_nodejs {
     npm install npm -g
 }
 
+# install_python() - Install Python 2.7 and other tools necessary for development.
+function install_python {
+    install_package python2.7
+    install_package python-dev
+    install_package python-pip
```
Comment (electrocucaracha) [inline lib/functions]: I prefer to install the latest version

    curl -sL https://bootstrap.pypa.io/get-pip.py | python

File: lib/functions [hunk 3]
```diff
@@ -118,6 +118,14 @@ function install_nodejs {
     npm install npm -g
 }
 
+# install_python() - Install Python 2.7 and other tools necessary for development.
+function install_python {
+    install_package python2.7
+    install_package python-dev
```
Comment (electrocucaracha) [inline lib/functions:124]: Do you need the python-dev package? if you need it, you will need to add an assertion into the UT.

    # apt-cache show python-dev
    Package: python-dev
    ...
    Description-en: header files and a static library for Python (default)
     Header files, a static library and development tools for building
     Python modules, extending the Python interpreter or embedding Python
     in applications.
     .
     This package is a dependency package, which depends on Debian's default
     Python version (currently v2.7).

File: lib/vfc [hunk 4]
```diff
@@ -2,7 +2,9 @@
 
 set -o xtrace
 
+source /var/onap/functions
 src_folder=/opt/vfc
```
Comment (electrocucaracha) [inline lib/vfc]: You can change it to:
    src_folder=$git_src_folder/vfc

File: lib/functions (hunk 3 above)
Comment (shank7485) [reply lib/functions:124]: Done.

File: lib/functions (hunk 2 above)
Comment (shank7485) [reply lib/functions]: Done. Added

File: lib/vfc (hunk 4 above)
Comment (shank7485) [reply lib/vfc]: Done.

File: tests/test_functions (hunk 1 above)
Comment (shank7485) [reply tests/test_functions]: Done. Added

**Review by shank7485 — COMMENTED** [review]
Done.

**Pull request merged**